### ISSUE TITLE
Determination of coordination and support for performance and measurement reporting within VHA (cont)

### ISSUE THREAD

**Issue by vvenu-va** [jamesmrollins/CIDMO_LCMT#52]
- [x] Finish determining which offices within VHA will be able to provide useful data for value stream metrics.   Specifically, what are they reporting?  What applies to CIDMO?  What may we get access to?  What level of support can be provided to CIDMO?
- [ ] Coordinate with IPEC
- [ ] ~~Coordinate with CSDE~~
- [x] Create instructions for navigating MCAO data

**Comment by vvenu-va** [comment]
On 22 March:
V presented the Managerial Accounting Pyramid that allows drill downs to various cost accounting factors.  He Demonstrated how to navigate to cost factors by department.

V and James discussed a use case of a specialty team who is describing a workflow in a given clinic and would need a cost factor to gauge the economic value of the project (the numerator in the value equation).

V and James also discussed putting together instructions to gain access to the Pyramid repo and do a drill down.

**Comment by vvenu-va** [comment]
[Instructions for Using Pyramid Cubes for MCAO Data.pdf](https://github.com/jamesmrollins/CIDMO_LCMT/files/8336305/Instructions.for.Using.Pyramid.Cubes.for.MCAO.Data.pdf)

**Comment by vvenu-va** [comment]
[Importand Sites for MCAO Data.docx](https://github.com/jamesmrollins/CIDMO_LCMT/files/8336312/Importand.Sites.for.MCAO.Data.docx)

**Comment by maneyb1** [comment]
Posted these files to LCMT Channel
@jamesmrollins, @brandymorrison - do review of instructions

**Comment by jamesmrollins** [comment]
Hi @vvenu-va - I reviewed the training documentation and was able to get into the Pyramid scheme (forgive the dad joke!).   The links did not survive the PDF export.  I was able to figure it out but may need to expand on which domain to select (MCAO).  

@maneyb1 one big limitation is that someone has to train up on Pyramid.  It is very similar to Power BI, with respect to grabbing information from a data base, slicing and dicing it, and presenting the information in various forms.  In short, it is going to be a lot of work getting familiar with the data universe of the CDW and then pulling in what we need. Pursuant to some of V's comments earlier in the week, we should look at the JD for his position to see what it has with respect to these skills (if lets us).
